### PR TITLE
#33: Code cleanup

### PR DESCRIPTION
**Pull request by rishabh-arya95**
# Description

Cleanup of unused code in js runners

Fixes # (issue)
Closes https://github.com/LambdaTest/test-at-scale-js/issues/33

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Tested all 3 runners individually.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/jest-runner/package.json [hunk 1]
```diff
@@ -28,6 +28,6 @@
   },
   "peerDependencies": {
     "@types/jest": "^27.0.1",
-    "jest": ">=21.2.1"
+    "jest": "^27.0.1"
```
Comment (saurabh-prakash) [inline packages/jest-runner/package.json:31]: remove this change

**Review by saurabh-prakash — COMMENTED** [review]
Please resolve conflicts.

File: packages/jest-runner/package.json (hunk 1 above)
Comment (rishabh-arya95) [reply packages/jest-runner/package.json]: done

**Review by saurabh-prakash — APPROVED** [review]

**Review by palashmarantas — APPROVED** [review]

File: packages/mocha-runner/src/mocha-runner.ts [hunk 2]
```diff
@@ -22,7 +22,6 @@ import {
     TestSuiteResult,
     Util,
     Validations,
-    InputConfig
```
Comment (palashmarantas) [inline packages/mocha-runner/src/mocha-runner.ts:25]: we did not removed this in neuron

**Review by palashmarantas — COMMENTED** [review]
Input config Did not removed in neuron. Did u tried running tests end to end

**Pull request merged**